agent/config/config_test.go

```diff
@@ -124,41 +110,33 @@ func TestEnvironmentConfig(t *testing.T) {
 	assert.Equal(t, (90 * time.Second), conf.TaskCleanupWaitDuration)
 }
 
-func TestTrimWhitespace(t *testing.T) {
+func TestTrimWhitespaceWhenCreating(t *testing.T) {
+	os.Setenv("AWS_DEFAULT_REGION", "us-west-2")
```

Comment:
Why was this added?

Reply:
Which one?

* `TestTrimWhitespace` was testing two different things, so I split it up.
* `os.Setenv` is there because the test was getting mad without the region explicitly set

Reply:
Why does it require `os.Setenv` now?

Reply:
I looked. The tests are depending on envs being set in other tests. I don't like that. I'll fix it.